src/main/java/website/frontrow/board/Bubble.java

```diff
@@ -15,9 +17,23 @@
 {
```

Comment:
Please place extends and implements on the same indentation level. I know you didn't do it, but it's nicer :)